--- old
+++ new
@@ -16,0 +16,6 @@
+É um dos elementos da tríade dos [[20260306181339|sistemas autorreferenciais]]:
+
+- [[20260306181315|Autopoiese]]
+- [[20260306181359|Fechamento operacional]]
+- [[20260306174301|Acoplamento estrutural]]
+
@@ -22,1 +28,0 @@
----
@@ -24,1 +29,0 @@
-## Conexões
@@ -26,1 +30,0 @@
-- [[20260306181339|Sistemas autorreferenciais]]: é um dos elementos da tríade
@@ -32,3 +35,0 @@
-
-
-