--- old
+++ new
@@ -13,1 +13,1 @@
-É essa circularidade que dá autonomia ao sistema, pois este decide o que é válido dentro de sua própria lógica. Quando um sistema observa o mundo, não consegue ver a lógica que está usando no momento da observação (ponto cego), pois esta é inerente.
+É essa circularidade que dá autonomia ao sistema, pois este decide o que é válido dentro de sua própria lógica. Quando um sistema observa o mundo, não consegue ver a lógica que está usando no momento da [[20260323085754|observação]] (ponto cego), pois esta é inerente.