--- old
+++ new
@@ -8,1 +8,1 @@
-Um [[20260307105528|sistema]] é autorreferencial quando usa a si mesmo como ponto de partida e de chegada para suas próprias operações. Ele é surdo a qualquer lógica que não seja a sua.
+Um [[20260307105528|sistema]] é [[20260308200131|autorreferencial]] quando usa a si mesmo como ponto de partida e de chegada para suas próprias operações. Ele é surdo a qualquer lógica que não seja a sua.