--- old
+++ new
@@ -11,1 +11,1 @@
-​O indivíduo é infinitamente [[20260309221929|complexo]] e instável. Ele sente fome, amor, tédio e muda de ideia.
+O indivíduo é infinitamente [[20260309221929|complexo]] e instável. Ele sente fome, amor, tédio e muda de ideia.
@@ -13,1 +13,1 @@
-​A [[20260309221929|complexidade]] do indivíduo é tão grande que se a [[20260314143114|sociedade]] tivesse que lidar com cada um não haveria [[20260309221624|comunicação]] possível. 
+A [[20260309221929|complexidade]] do indivíduo é tão grande que se a [[20260314143114|sociedade]] tivesse que lidar com cada um não haveria [[20260309221624|comunicação]] possível. 