--- old
+++ new
@@ -9,1 +9,1 @@
-Alopoiese é o processo pelo qual um sistema produz um resultado externo a si mesmo, em vez de [[20260306181315|reproduzir]] seus próprios elementos. Ou seja, o produto do sistema não é o próprio sistema. Isso é o oposto à definição de [[20260307105528|sistema autopoiético]].
+Alopoiese é o processo pelo qual um sistema produz um resultado externo a si mesmo, em vez de [[20260306181315|reproduzir]] seus próprios elementos. Ou seja, o produto do sistema não é o próprio sistema. Isso é o oposto à definição de [[20260307105528|sistema autopoiético]] [[20260306181315|]].