--- old
+++ new
@@ -14,1 +14,1 @@
-O sistema permite ser [[20260306174301|irritado]] (ou estimulado) pelo que acontece fora. Sem a heterorreferência o sistema não teria "assunto". O Direito, por exemplo, ficaria repetindo leis sem nunca aplicá-las a conflitos sociais reais.
+O sistema permite ser [[20260306174301|irritado]] (ou estimulado) pelo que acontece fora. Sem a heterorreferência o sistema não teria assunto. O Direito, por exemplo, ficaria repetindo leis sem nunca aplicá-las a conflitos sociais reais.