--- old
+++ new
@@ -11,2 +11,1 @@
-Em sistemas sociais uma comunicação só gera outra comunicação.
-Um pensamento precisa ser traduzido em algo que o sistema entenda (palavras, gestos, sinais) para que possa ser transmitido de um indivíduo a outro.
+Em sistemas sociais uma comunicação só gera outra comunicação. Um pensamento precisa ser traduzido em algo que o sistema entenda (palavras, gestos, sinais) para que possa ser transmitido de um indivíduo a outro.