--- old
+++ new
@@ -19,0 +19,4 @@
+A realidade ontológica é uma construção simplificadora que permite ao observador ignorar sua própria participação na criação daquilo que ele vê. Para a teoria dos sistemas o "Ser" é substituído pelo sentido e pela diferença. Não importa o que a coisa "é", mas sim como ela se diferencia do seu entorno e qual função ela cumpre dentro de uma rede de comunicações.
+
+A realidade não pode ser compreendida objetivamente sem que haja o viés de um observador.
+