--- old
+++ new
@@ -5,1 +5,1 @@
-O ser humano não faz parte de sistemas sociais. Ele os irrita como uma forma de acoplamento estrutural. A sociedade é feita de comunicações.
+O ser humano não faz parte de sistemas sociais. Ele os irrita como uma forma de acoplamento estrutural.
@@ -7,0 +7,1 @@
+A sociedade não é formada por indivíduos, mas em comunicações; e o indivíduo é apenas o ponto de intersecção de diversos sistemas sociais.
@@ -8,0 +9,1 @@
+