--- old
+++ new
@@ -11,1 +11,1 @@
-Poiesis refere-se à criação ou fabricação. É uma ação voltada para produzir algo. Ao contrário de práxis, que é uma ação que tem o seu fim em si mesma, na excelência da própria ação e o aperfeiçoamento de quem a pratica.
+Poiesis refere-se à criação ou fabricação, uma ação voltada para produzir algo. Ao contrário da práxis, que é uma ação que tem o seu fim em si mesma, na excelência da própria ação e o aperfeiçoamento de quem a pratica.
@@ -14,1 +14,1 @@
- entre o sistema e o ambiente. 
+ entre o sistema e o ambiente [@NiklasLuhmann434, p. 78]. 