--- old
+++ new
@@ -18,1 +18,1 @@
-Os elementos do sistema (comunicações ou pensamentos) são eventos efêmeros e desaparecem no instante em que ocorrem. O sistema é forçado a produzir uma nova operação o tempo todo para não deixar de existir. A produção desses elementos é, portanto, uma corrida constante contra o fim do sistema.
+Os elementos do sistema ([[20260309221624|comunicações]] ou pensamentos) são eventos efêmeros e desaparecem no instante em que ocorrem. O sistema é forçado a produzir uma nova operação o tempo todo para não deixar de existir. A produção desses elementos é, portanto, uma corrida constante contra o fim do sistema.