--- old
+++ new
@@ -16,1 +16,1 @@
-Forma pela qual a comunicação se organiza em sistemas autônomos (Direito, Economia, Ciência), que se mantêm vivos independentemente das intenções individuais dos seres humanos. 
+Forma pela qual a [[20260309221624|comunicação]] se organiza em sistemas autônomos (Direito, Economia, Ciência), que se mantêm vivos independentemente das intenções individuais dos seres humanos. 