--- old
+++ new
@@ -21,0 +21,6 @@
+É um dos elementos da tríade dos [[20260306181339|sistemas autorreferenciais]]:
+
+- [[20260306181315|Autopoiese]]
+- [[20260306181359|Fechamento operacional]]
+- [[20260306174301|Acoplamento estrutural]]
+
@@ -27,1 +33,0 @@
----
@@ -29,1 +34,0 @@
-## Conexões
@@ -31,1 +35,0 @@
-- [[20260306181339|Sistemas autorreferenciais]]: é um dos elementos da tríade.
@@ -33,3 +36,0 @@
-
-
-