--- old
+++ new
@@ -8,1 +8,1 @@
-O que se chama de "conhecimento" não é o encontro de um sujeito com um objeto como o preconizado pela filosofia clássica, mas sim o resultado de duas formas de referência que o [[20260306181339|sistema]] realiza simultaneamente: a autorreferência e a heterorreferência. 
+O que se chama de "conhecimento" não é o encontro de um sujeito com um objeto como o preconizado pela filosofia clássica, mas sim o resultado de duas formas de referência que o [[20260306181339|sistema]] realiza simultaneamente: a [[20260308200131|autorreferência]] e a heterorreferência. 
@@ -12,1 +12,1 @@
-- Autorreferência: relação consigo mesmo
+-  [[20260308200131|Autorreferência]] : relação consigo mesmo
@@ -15,1 +15,1 @@
-Não há a ideia de um sujeito que conhece o mundo, mas sim de sistemas que observam. Cada sistema constrói suas próprias referências.
+Não há a ideia de um sujeito que conhece o mundo e de um objeto que é conhecido. Há sistemas que observam, cada um construindo suas próprias referências.