--- old
+++ new
@@ -8,1 +8,1 @@
-O que se chama de "conhecimento" não é o encontro de um sujeito com um objeto como o preconizado pela filosofia clássica, mas sim o resultado de duas formas de referência que o [[20260306181339|]] realiza simultaneamente: a autorreferência e a heterorreferência. 
+O que se chama de "conhecimento" não é o encontro de um sujeito com um objeto como o preconizado pela filosofia clássica, mas sim o resultado de duas formas de referência que o [[20260306181339|sistema]] realiza simultaneamente: a autorreferência e a heterorreferência. 