--- old
+++ new
@@ -18,0 +18,6 @@
+É um dos elementos da tríade dos [[20260306181339|sistemas autorreferenciais]]:
+
+- [[20260306181315|Autopoiese]]
+- [[20260306181359|Fechamento operacional]]
+- [[20260306174301|Acoplamento estrutural]]
+
@@ -24,1 +30,0 @@
----
@@ -26,1 +31,0 @@
-## Conexões
@@ -28,1 +32,0 @@
-- [[20260306181339|Sistemas autorreferenciais]]: é um dos elementos da tríade
@@ -30,1 +33,0 @@
-