--- old
+++ new
@@ -11,1 +11,1 @@
-Forma pela qual a comunicação se organiza em sistemas autônomos (Direito, Economia, Ciência), que se mantêm vivos independentemente das intenções individuais dos seres humanos. 
+Poiesis refere-se à criação ou fabricação, uma ação voltada para produzir algo. Ao contrário da práxis, que é uma ação que tem o seu fim em si mesma, na excelência da própria ação e o aperfeiçoamento de quem a pratica.
@@ -13,1 +13,2 @@
-Os elementos do sistema (comunicações ou pensamentos) são eventos efêmeros e desaparecem no instante em que ocorrem. O sistema é forçado a produzir uma nova operação o tempo todo para não deixar de existir. A produção desses elementos é, portanto, uma corrida constante contra o fim do sistema.
+A obra produzida é o próprio sistema, ou a forma do sistema, na [[20260308201523 | diferença]] 
+ entre o sistema e o ambiente [@NiklasLuhmann434, p. 78]. 
@@ -15,0 +16,4 @@
+Forma pela qual a [[20260309221624|comunicação]] se organiza em sistemas autônomos (Direito, Economia, Ciência), que se mantêm vivos independentemente das intenções individuais dos seres humanos. 
+
+Os elementos do sistema ([[20260309221624|comunicações]] ou pensamentos) são eventos efêmeros e desaparecem no instante em que ocorrem. O sistema é forçado a produzir uma nova operação o tempo todo para não deixar de existir. A produção desses elementos é, portanto, uma corrida constante contra o fim do sistema.
+