--- old
+++ new
@@ -15,1 +15,1 @@
-- meios de comunicação simbolicamente generalizados. Utilizados pelos sistemas funcionais: verdade (ciência), dinheiro (economia), poder (política), lei (Direito), arte (arte), fé (religião), etc. 
+- meios de comunicação simbolicamente generalizados. Utilizados pelos sistemas funcionais: verdade (Ciência), dinheiro e propriedade (Economia), poder (Política), lei (Direito), fé (Religião), etc. 