--- old
+++ new
@@ -9,1 +9,1 @@
-​Se o sistema parasse de olhar para si mesmo, ele seria "engolido" pelo ambiente e deixaria de ter uma identidade própria.
+Se o sistema parasse de olhar para si mesmo, ele seria "engolido" pelo ambiente e deixaria de ter uma identidade própria.
@@ -11,1 +11,1 @@
-​A operação atual só acontece porque se baseia em operações passadas do próprio sistema, num processo circular de [[20260306181315|autopoiese]].
+A operação atual só acontece porque se baseia em operações passadas do próprio sistema, num processo circular de [[20260306181315|autopoiese]].
@@ -13,1 +13,1 @@
-​Essa circularidade que dá autonomia ao sistema, pois este decide o que é válido dentro de sua própria lógica.
+Essa circularidade que dá autonomia ao sistema, pois este decide o que é válido dentro de sua própria lógica.