--- old
+++ new
@@ -8,1 +8,1 @@
-​A pessoa (Artefato Social) é um endereço de [[20260309221624|comunicação]]. É uma construção que o [[20260307105528|sistema social]] cria para simplificar a [[20260309221929|complexidade]] do indivíduo.
+A pessoa (Artefato Social) é um endereço de [[20260309221624|comunicação]]. É uma construção que o [[20260307105528|sistema social]] cria para simplificar a [[20260309221929|complexidade]] do indivíduo.
@@ -10,2 +10,2 @@
-​Serve para que a [[20260314143114|sociedade]] possa atribuir responsabilidades, expectativas e direitos. A sociedade não quer saber da  vida interior do [[20260323092108|indivíduo]], pois ela precisa apenas de um ponto de referência. Por exemplo: "esta pessoa assinou o contrato" ou "esta pessoa cometeu um crime".
-​
+Serve para que a [[20260314143114|sociedade]] possa atribuir responsabilidades, expectativas e direitos. A sociedade não quer saber da  vida interior do [[20260323092108|indivíduo]], pois ela precisa apenas de um ponto de referência. Por exemplo: "esta pessoa assinou o contrato" ou "esta pessoa cometeu um crime".
+