--- old
+++ new
@@ -8,1 +8,1 @@
-A pessoa (Artefato Social) é um endereço de [[20260309221624|comunicação]]. É uma construção que o [[20260307105528|sistema social]] cria para simplificar a [[20260309221929|complexidade]] do indivíduo.
+A pessoa é um endereço de [[20260309221624|comunicação]]. É uma construção que o [[20260307105528|sistema social]] cria para simplificar a [[20260309221929|complexidade]] do indivíduo.