--- old
+++ new
@@ -8,1 +8,1 @@
-Um sistema é um conjunto [[20260306181315|autopoiético]] de operações que produz e reproduz continuamente a distinção entre sistema e ambiente. É uma diferença em relação ao ambiente que se mantém através de suas próprias operações.
+Um sistema social é um conjunto [[20260306181315|autopoiético]] de operações que produz e reproduz continuamente a distinção entre sistema e ambiente. É uma diferença em relação ao ambiente que se mantém através de suas próprias operações.