--- old
+++ new
@@ -9,2 +9,1 @@
-Há outras possibilidades 
----
+Escolher uma possibilidade dentre outras disponíveis.
@@ -12,1 +11,1 @@
-## Referências
+O significado dos códigos binários pode variar de acordo com a sociedade e com a época.
@@ -14,1 +13,0 @@
-- [[link_da_citacao]]: Por que essa referência é relevante
@@ -16,1 +14,0 @@
----
@@ -18,4 +15,0 @@
-## Conexões
-
-- [[ID]]: Explicação de como a nota se conecta à outra
-