--- old
+++ new
@@ -16,1 +16,1 @@
-**Compreensão**: É a seleção final feita pelo sistema. Tem como fundamento a percepção que há uma diferença entre a informação e o ato de comunicar. Significa que o sistema social processou a distinção entre o conteúdo e o ato. Não tem relação com o entendimento. Mesmo que o entendimento tenha sido incorrero, houve comunicação.
+**Compreensão**: É a seleção final feita pelo sistema. Tem como fundamento a percepção que há uma diferença entre a informação e o ato de comunicar. Significa que o sistema social processou a distinção entre o conteúdo e o ato. Não tem relação com o entendimento, aceitação ou recusa. Mesmo que tenha havido recusa ou que o entendimento tenha sido incorreto, houve comunicação.